### ISSUE TITLE
Possibly improve OneCore voices' quality when used with NVDA

### ISSUE THREAD

**Issue by Cleversn** [nvaccess/nvda#11649]
### Is your feature request related to a problem? Please describe.
A friend told me that, when used with JAWS, the OneCore voices sound higher and clearer.
### Describe the solution you'd like
You might investigate whether recent Windows updates introduced enhancements to the OneCore voices, and if so, enhance the NVDA interaction with them.
### Describe alternatives you've considered
None.
### Additional context
None.

**Comment by josephsl** [comment]
Hi, it might be possible to look into updating local Win10 lib to account for API updates. Another possibility might be a confusion between SAPI5 and OneCore voices. CC @MichaelDCurran, @JcSTeh if he has any ideas. Thanks.

**Comment by leonardder** [comment]
It would help if we have a comparison recording to hear the proof of this point.
See also #10662

**Comment by codeofdusk** [comment]
Are the higher-quality voices available in Narrator?

**Comment by Mohamed00** [comment]
As far as I can tell, there are no differences in OneCore when used in Narrator and NVDA, so I'm not sure what this issue is referring to.

**Comment by bhavyashah** [comment]
I am unable to observe any noticeable difference in the quality of OneCore voices between NVDA and Narrator. I would note that my testing was limited to Microsoft David. A few questions for @Cleversn:
* Is this relatively superior handling of OneCore voices unique to JAWS, or true of Narrator as well?
* Which Microsoft OneCore Voice(s) is/are being used when experiencing this?
* Is this something you are able to independently verify yourself as well?

**Comment by seanbudd** [comment]
Considering that multiple people have confirmed that there are no noticeable differences between the voices, I am closing this issue. This will be reopened if it can be shown to still be an issue, with answers to @bhavyashah's questions.
